lib/command/src/Obelisk/Command/VmBuilder.hs

```diff
@@ -133,13 +152,18 @@ getDockerBuilderStateDir :: IO FilePath
 getDockerBuilderStateDir = liftA2 (</>) getObeliskUserStateDir (pure "nix-docker-builder")
 
 -- | Run a test build to see if a Linux build can finish successfully.
-testLinuxBuild :: MonadObelisk m => FilePath -> m Bool
-testLinuxBuild stateDir =
-  fmap (isRight :: Either ExitCode () -> Bool) $ try $ withNixRemoteCheck $
-  callProcessAndLogOutput (Debug, Debug) $ proc "nix-build"
-    [ "-E", "(import <nixpkgs> { system = \"x86_64-linux\"; }).writeText \"test\" builtins.currentTime"
-    , "--builders", nixBuildersArgString stateDir
-    ]
+testLinuxBuild :: MonadObelisk m => FilePath -> m ()
+testLinuxBuild stateDir
+  | System.Info.os == "linux" = failWith "Using the docker builder is not necessary on linux."
```

Comment:
This shouldn't be necessary since the command-line parser auto defaults to turning off VM builder on Linux. If they explicitly set it, then I don't see a reason to prevent them from using a remote builder. However, it's now questionable weather the `--builder` flag is the correct way to do this on newer nix so it may be moot.